src/wasm-linker.cpp

```diff
@@ -362,9 +374,26 @@ void Linker::makeDynCallThunks() {
     for (unsigned i = 0; i < funcType->params.size(); ++i) {
       args.push_back(wasmBuilder.makeGetLocal(i + 1, funcType->params[i]));
     }
-    Expression* call = wasmBuilder.makeCallIndirect(funcType, fptr, std::move(args));
+    Expression* call = wasmBuilder.makeCallIndirect(funcType, fptr, args);
     f->body = call;
     out.wasm.addFunction(f);
     exportFunction(f->name, true);
   }
 }
+
+Function* Linker::generateImportThunk(Name name, const FunctionType* funcType) {
+  wasm::Builder wasmBuilder(out.wasm);
+  std::vector<NameType> params;
+  int p = 0;
```

Comment:
could be `Index` (could matter in the future, when wasm64 has 64-bit Index, and the person is building on a 32-bit machine)